### PR TITLE
Feature 5 make footer responsible

### PR DESCRIPTION
**Pull request by luciagirasoles**
- Add tailwind to the project
- Update some html tags
- Make footer responsive
Rel #5

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ gem 'rails', '~> 6.1.4'
 gem 'pg'
 gem 'puma', '~> 5.0'
 gem 'sass-rails', '>= 6'
-gem 'webpacker', '~> 5.0'
+gem 'webpacker', '~> 5.4.0'
```
Comment (egiurleo) [inline Gemfile:12]: Was this change necessary for using Tailwind? (Asking to educate myself on dependencies 😃)

**Review by egiurleo — COMMENTED** [review]
@luciagirasoles Awesome job adding Tailwind and making the footer responsive! I left one question and I also noticed that this made some changes to the styles of the footer on a larger screen (see the screenshot below). The footer now has more margin than the header, and the social links don't line up in a row. Would you mind fixing these two issues on larger screen sizes? Then I'd be happy to approve and merge this PR.

Screen Shot 2021-10-05 at 3 14 10 PM

File: Gemfile (hunk 1 above)
Comment (luciagirasoles) [reply Gemfile:12]: The problem with the `5.0` version is that includes PostCSS 7 but the latest version of Tailwind requires version 8. So I had to follow this article: https://dev.to/davidteren/getting-started-with-rails-6-1-tailwindcss-jit-webpacker-postcss-8-25g7
